DeploymentTomcat.md

```diff
@@ -12,6 +12,13 @@ This page describes how to implement the latter option and has been validated us
 
 Once you have built the application following the steps outlined in the [Developer's Guide](DevelopersGuide.md), execute the following steps to deploy your application to Tomcat.
 
+---
+**NOTE**
```

Comment:
Looks good. FYI, I had already added a question and answer to the FAQ. See here: https://github.com/attivio/searchui/blob/master/FAQ.md#how-do-i-run-two-servlet-hosted-search-uis-on-the-same-machine

(I was waiting for feedback from Ben Vachon before sending around a note about it...)